### ISSUE TITLE
Addd a tooltip on Timer/Stopwatch indicator

### ISSUE THREAD

**Issue by antobinary** [bigbluebutton/bigbluebutton#18442]
**Describe the bug**
Hovering over the indicator does not bring up a tooltip. This may also be an indicator of missing accessibility compliance.

**To Reproduce**
Hover over 
![Screenshot from 2023-08-03 09-46-47](https://github.com/bigbluebutton/bigbluebutton/assets/6312397/692c0522-1c17-4a64-b225-2000a80655d9)

**Expected behavior**
Tooltip with text "The timer has stopped, there are 10 seconds left" or "The timer has stopped, it reads 00:00:10s" if it was a stopwatch the text could be "The stopwatch was paused at 00:00:10s". Check what we do with the meeting recorder timer, possibly there is accessibility code for not disturbing on every second...

**Actual behavior**
No tooltip.

**Screenshots**
above

**BBB version:**
BigBlueButton 2.7.0-beta.3

**Additional context**
